ui/guest/src/controls/rte.ts

```diff
@@ -42,23 +42,34 @@ export function initTinyMCE(
   const { field, model } = iceRegistry.getReferentialEntries(record.iceIds[0]);
   const type = field?.type;
   const inlineElsRegex =
-    /B|BIG|I|SMALL|TT|ABBR|ACRINYM|CITE|CODE|DFN|EM|KBD|STRONG|SAMP|VAR|A|BDO|BR|IMG|MAP|OBJECT|Q|SCRIPT|SPAN|SUB|SUP|BUTTON|INPUT|LABEL|SELECT|TEXTAREA/;
+    /^B$|^BIG$|^I$|^SMALL$|^TT$|^ABBR$|^ACRINYM$|^CITE$|^CODE$|^DFN$|^EM$|^KBD$|^STRONG$|^SAMP$|^VAR$|^A$|^BDO$|^BR$|^IMG$|^MAP$|^OBJECT$|^Q$|^SCRIPT$|^SPAN$|^SUB$|^SUP$|^BUTTON$|^INPUT$|^LABEL$|^SELECT$|^TEXTAREA$/;
   let rteEl = record.element;
   const isRecordElInline = record.element.tagName.match(inlineElsRegex);
 
   // If record element is of type inline (doesn't matter the display prop), replace it with a block element (div).
   if (isRecordElInline) {
+    const recordEl = record.element;
     const blockEl = document.createElement('div');
-    blockEl.innerHTML = record.element.innerHTML;
+    blockEl.innerHTML = recordEl.innerHTML;
+
+    // Copy original element inline styles (copying all styles causes issues with XB styles because they end up all
+    // being inline styles).
```

Comment:
What are "all styles"? Isn't that what you're doing? What's the problem with "all ending up as inline"?

Reply:
Ok. That 👆🏻 should be the code comment, just clean it up slightly for conciseness, but it's ok to write multiline comments that better explain things.